cukinia

```diff
@@ -891,6 +897,66 @@ _cukinia_knoerror()
 	return $ret
 }
 
+# _cukinia_network_interface_up: check if a network interface is up
+# arg1: interface name
+_cukinia_network_interface_up() {
+	local interface="$1"
+
+	_cukinia_prepare "Checking if interface \"$interface\" is ${__not:+NOT }up"
+
+	ip link show "$interface" up | grep -q 'state UP' >/dev/null 2>&1 && return 0 || return 1
```

Comment:
double quotes on "state UP" for consistency

Reply:
Done